checkstyle-tester/diff.groovy

```diff
@@ -373,6 +385,46 @@ def cloneRepository(repoName, repoType, repoUrl, commitId, srcDir) {
     println "$repoName is synchronized"
 }
 
+def getCloneCmd(repoType, repoUrl, srcDestinationDir) {
+    final String cloneCmd = ''
+    switch (repoType) {
```

Comment:
Please follow the same pattern as `getCloneShallowCmd`, using an if statement instead of switch for a single case.

Reply:
Done.